teuthology/task/install/__init__.py

```diff
@@ -356,7 +362,7 @@ def upgrade_common(ctx, config, deploy_style):
         pkgs += extra_pkgs
 
         installed_version = packaging.get_package_version(remote, 'ceph-common')
-        upgrade_version = _get_gitbuilder_project(ctx, remote, node).version
+        upgrade_version = get_upgrade_version(ctx, remote, node)
```

Comment:
@jan--f: These improvements seem like they would rather be in the first commit. Would it be a good idea to move them there (and add a `Signed-off-line:` with your name)?